crates/re_data_store/src/entity_tree.rs

```diff
@@ -69,8 +50,31 @@ pub struct EntityTree {
     pub time_histograms_per_component: BTreeMap<ComponentName, TimeHistogramPerTimeline>,
 }
 
+impl StoreView for EntityTree {
+    fn name(&self) -> String {
+        "rerun.store_views.EntityTree".into()
+    }
+
+    fn as_any(&self) -> &dyn std::any::Any {
+        self
+    }
+
+    fn as_any_mut(&mut self) -> &mut dyn std::any::Any {
+        self
+    }
+
+    #[allow(clippy::unimplemented)]
+    fn on_events(&mut self, _events: &[StoreEvent]) {
+        unimplemented!(
+            r"EntityTree view is maintained manually, see `EntityTree::on_store_{{additions|deletions}}`"
+        );
+    }
```

Comment:
Similar to the last review -- what use is implementing `StoreView` if we don't support `on_events`?

Reply:
See https://github.com/rerun-io/rerun/pull/4208#discussion_r1393968207